### PR TITLE
Excalidraw Embeds

### PR DESCRIPTION
**Pull request by j-f1**
During the build process, it opens the links in Excalidraw using Puppeteer, then exports them as SVG and injects that directly into the blog post.

Currently, these are missing alt text. Maybe the export file name could be included in the encrypted blob when you save a drawing, and we could turn it into the alt text?

### CODE REVIEW

**Comment by lipis** [comment]
Green!

**Comment by j-f1** [comment]
Another thing is that the `<style>` is repeated with each SVG, but we could punt that issue and the one about alt text to a later date.

**Comment by Fausto95** [comment]
Awesome

**Comment by vjeux** [comment]
Since this was merged, I can't build the site anymore :( https://github.com/gatsbyjs/gatsby/issues/23033

File: src/excalidraw-embed/get-image.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  const frame = await page.mainFrame();
+  const result = await frame.evaluate(
+    () =>
+      new Promise((resolve) => {
+        const button = document.querySelector('[aria-label="Export to SVG"]');
+        const link = document.createElement("a");
+        delete window.chooseFileSystemEntries;
+        document.createElement = () => link;
+        link.click = () =>
+          fetch(link.href)
```
Comment (vjeux) [inline src/excalidraw-embed/get-image.js:21]: This is failing with "TypeError: Failed to fetch" where link.href is `blob:https://excalidraw.com/ab7b2f84-9207-4ee5-83c5-eaa94c0f969c`

**Pull request merged**